"grep [A-z]" is not allowed in non-C locale.
---
 util/texi2dvi |    2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
  
Index: texinfo-4.12.dfsg.1/util/texi2dvi
===================================================================
--- texinfo-4.13a.dfsg.1.orig/util/texi2dvi.orig	2010-11-04 15:28:50.000000000 +0100
+++ texinfo-4.13a.dfsg.1/util/texi2dvi	2010-11-04 15:35:14.000000000 +0100
@@ -1710,7 +1710,7 @@
 
   # If the COMMAND_LINE_FILENAME is not absolute (e.g., --debug.tex),
   # prepend `./' in order to avoid that the tools take it as an option.
-  echo "$command_line_filename" | $EGREP '^(/|[A-z]:/)' >&6 \
+  echo "$command_line_filename" | $EGREP '^(/|[A-Za-z]:/)' >&6 \
   || command_line_filename="./$command_line_filename"
 
   # See if the file exists.  If it doesn't we're in trouble since, even
